LSLib/LS/PackageCommon.cs

```diff
@@ -68,7 +68,7 @@ internal struct FileEntry13
         public UInt32 Crc;
     }
 
-    abstract public class FileInfo
+    abstract public class CommonPackageInfo
```

Comment:
`AbstractFileInfo` would probably be more appropriate, The name `CommonPackageInfo` implies that the class contains some shared information about a package, but in fact it is a wrapper around packaged or unpackaged files.